Tools/CISupport/ews-build/steps.py

```diff
@@ -6683,9 +6683,9 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
         self.commands = []
         commands = [
             f"git log {head_ref} ^{base_ref} | grep -q '{self.OOPS_RE}' && echo 'Commit message contains (OOPS!){reviewer_error_msg}' || test $? -eq 1",
-            "git log {} ^{} | grep -q '\\({}\\)' || echo 'No reviewer information in commit message'".format(
+            "TMPFILE=$(mktemp); git log {} ^{} > '$TMPFILE'; grep -q '\\({}\\)' '$TMPFILE' || echo 'No reviewer information in commit message'; rm -f '$TMPFILE'".format(
```

Comment:
`'$TMPFILE'` with single quotes is not going to expand the variable, so I suspect this code writes out an actual file named "$TMPFILE".

…honestly, it might be fine to do that, using a more obvious file name like `COMMIT_MSG`? That would make it possible to observe what this step is doing on the worker if there's ever an issue. With that in mind, it might also be appropriate to ditch the `rm -f` and keep the file around until its overridden by the next build.